playbooks/roles/ocp-config/defaults/main/main.yaml

```diff
@@ -17,3 +17,4 @@ cni_network_provider: OpenshiftSDN
 cluster_network_cidr: 10.128.0.0/14
 cluster_network_hostprefix: 23
 service_network: 172.30.0.0/16
+fips: "{{ fips_compliant }}"
```

Comment:
This does not seem proper to me, it should have some default value if applicable rather than deducing it.

Reply:
Updated required changes as per @Prajyot-Parab comments and also updated relevant docs, can you please review it again?